scripts/yggdrasil/install

```diff
@@ -43,19 +43,33 @@ fi
 sudo sed -i "s/IfName: auto/IfName: ygg0/" /etc/yggdrasil.conf 
 
 # Change port to 12345 for firewall
-YGGDRASIL_PORT=$(sudo cat /etc/yggdrasil.conf | grep \ Listen: | awk  '{print $2}' | tr -d \")
-YGGDRASIL_PORT=$(printf "%q" "$YGGDRASIL_PORT") # Escape for sed
-sudo sed  -i "s/$YGGDRASIL_PORT/\[::\]:12345/" /etc/yggdrasil.conf
+sudo sed -i "s/Listen: \[\]/Listen\: [\"tcp:\/\/\[::\]:12345\"\]/" /etc/yggdrasil.conf
 
+sudo systemctl enable yggdrasil.service
+sudo systemctl start yggdrasil.service 
 
-# Set node name
-if [ -z "${NEWHOSTNAME}" ]; then 
-    NEWHOSTNAME=$(cat /etc/hostname)
+# Dont count yggdrasill ips as internet ip addresses
+if [ -f "/etc/gai.conf" ]; then
```

Comment:
What is this file?

Reply:
Gia - getInternerAddress

c library to resolve names  

used to downgrade yggdrssil IPs so they are not considered internet ips

Reply:
Then this file and all the references to it should be changed to `gia.conf`. Thanks.

Reply:
> I would merge `develop` back into this and update the TOC after you make my changes.

One reason i want to just merge  this into develop is to clean up the docs a bit more

>     * I don't understand the distinction between `BOARD_NAME` and `BOARD_MODEL`, at least in the scripts you've written
`BOARD_NAME` - Human readable `Raspberry Pi 3B+` or `Orange Pi One`
`BOARD_MODEL` - raspberrypizero or orangepizeroplus2-h5